solr/solr-ref-guide/modules/indexing-guide/pages/indexing-nested-documents.adoc

```diff
@@ -304,6 +305,11 @@ Instead, use delete-by-query (most efficient) or xref:partial-document-updates.a
 If you use Solr's delete-by-query APIs, you *MUST* be careful to ensure that any deletion query is structured to ensure no descendent children remain of any documents that are being deleted.
 *_Doing otherwise will violate integrity assumptions that Solr expects._*
 
+=== Shard Splits
+When xref:deployment-guide:shard-management.adoc#splitshard[splitting shards], child documents get transmitted and routed on their own, since they are documents in their own right. If you route based on your unique field, then the child documents will be routed correctly based on their `_root_`. However, if your collection specifies a router field (even if it matches the unique field), then the child documents will be routed based on that field instead of their root field. If your collection specifies a router field, you should ensure every doc in a hierarchy has the same value for that field, otherwise it maybe broken during a shard split.
```

Comment:
Hmm; I suppose we might add such a tip in a number of places.  In Nested Docs, or in Shard Splits, or `collection-management.adoc` (where  router.field is documented).  I suppose here in this "maintaining integrity" in nested docs is okay but I also think it should be where `router.field` is defined to simply say that all docs, _even child docs_, need this field, and that all docs in a nested doc tree must have the same value.  Needn't list what bad things happen when you don't.

I suppose Solr should do that itself -- I filed: https://issues.apache.org/jira/browse/SOLR-17517

Reply:
Please put a newline at the end of each sentence in our adoc files.  It's then easier to handle diffs and it's less of a problem when an editor doesn't wrap.

Reply:
This paragraph feels more like it should be a warning/tip than something that needs its own heading.  And although your inside/internal information is interesting, I think it's better to be short & sweet get to the point of what a user must do:  router.field must be consistent for the nested doc tree.

Reply:
## Reworded, same formatting
![image](https://github.com/user-attachments/assets/1ed5091c-90bf-44b9-a91f-f3be7332430e)

## Caution formatting
(Copied from caution above in file...)
\-
![image](https://github.com/user-attachments/assets/6d8bd55e-3cc6-4bae-b1a6-942014ef0a25)

## Warning formatting
Any existing guidance on caution vs warning? 🤦 
\- 
![image](https://github.com/user-attachments/assets/fc421add-0f57-43e7-b02d-901d8e3aab89)

## Note formatting
🤔 🙃 
\-
![image](https://github.com/user-attachments/assets/2dbd4ace-bbf5-42ca-bec5-19c7f9c379eb)

Reply:
Hmm; I suppose this whole "Maintaining Integrity with Updates and Deletes" is a big fat warning, so maybe we don't need to single out this bit and say it's special.  A little paragraph on this is fine.

Reply:
Switched to humble little paragraph (with breadcrumb in title).
![image](https://github.com/user-attachments/assets/62b3c052-f2c3-461f-be3f-4c5314620a51)